src/entities/user.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserData {
```

Comment:
Actually, this is just a type, not an interface. In general, it doesn't make much difference, but still, interfaces provide some extra features and usually describe _interface_ (some class) :)